src/saltext/vmware/modules/vmc_org_users.py

```diff
@@ -0,0 +1,441 @@
+"""
+Salt execution module for VMC org users
+Provides methods to get, add and remove user within a org.
+"""
+import logging
+
+from saltext.vmware.utils import vmc_constants
+from saltext.vmware.utils import vmc_request
+from saltext.vmware.utils import vmc_templates
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = "vmc_org_users"
+__func_alias__ = {"list_": "list"}
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def list_(
+    hostname,
+    refresh_key,
+    org_id,
+    expand_profile=None,
+    include_group_ids_in_roles=None,
+    page_limit=None,
+    page_start=None,
+    service_definition_id=None,
+    verify_ssl=True,
+    cert=None,
+):
+    """
+    Returns list of users for the given org
+
+    Please refer the `Get Org Users documentation <https://developer.vmware.com/apis/csp/csp-iam/latest/csp/gateway/am/api/v2/orgs/orgId/users/get/>`_ to get insight of functionality and input parameters
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt <minion id> vmc_org_users.get hostname=console.cloud.vmware.com org_id=org_id  ...
+
+    hostname
+        The host name of CSP
+
+    refresh_key
+        API Token of the user which is used to get the Access Token required for VMC operations
+
+    org_id
+        The Id of organization for which user list is retrieved
+
+    expand_profile: String
+        (Optional) Indicates if the response should be expanded with the user profile, the value is ignored, only the existence of parameter is checked.
```

Comment:
**suggestion** it's OK if the underlying API doesn't care what the value is. We should. This should be bool with a default of False. Then in the code:

```
if expand_profile:
    prarams['expandProfile'] = 'yes'
```

Or, whatever.

Reply:
updated properly